repos/rust-lang/style-team.toml

```diff
@@ -0,0 +1,7 @@
+org = "rust-lang"
+name = "style-team"
+description = "Home of the Rust style team"
+bots = ["rustbot"]
+
+[access.teams]
+style = "maintain"
```

Comment:
Do you also want branch protection?

```
[[branch-protections]]
pattern = 'main'
```

Reply:
@rylev The original goal of this commit was to enable rustbot, and we don't at this time have any specific plans for using a bot to merge things, so we'd need to coordinate and discuss that before switching to it.

Reply:
FYI: the branch protection is also useful even if you're merging things manually - it prevents people pushing to main without code review.